languageservice/src/context-providers/events/eventPayloads.ts

```diff
@@ -1,7 +1,8 @@
 import {data, DescriptionDictionary} from "@actions/expressions";
 
```

Comment:
`eventsPayloads.ts` reads the compact webhook JSON files which contains the keys and descriptions for each GitHub event payload. This information is used to validate `github.event.*` syntax and provide descriptions on hover.

A unit test that runs during CI validates the compact webhook JSON files are parsed into a form that exactly matches the full unoptimized webhooks JSON.

The script `update-webhooks.ts` downloads the full webhook JSON from the GitHub REST API description repository:

```typescript
import schemaImport from "rest-api-description/descriptions/api.github.com/dereferenced/api.github.com.deref.json";
```